### PR TITLE
Add a Code of Conduct

### PR DESCRIPTION
**Pull request by fabianmoronzirfas**
I think we should have on (also if we want to keep on using Netlify we need one)

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at info@technologiestiftung-berlin.de. All
```
Comment (b-g) [inline CODE_OF_CONDUCT.md]: Wrong email address :)

**Comment by b-g** [comment]
@fabianmoronzirfas Many thanks for this! Just found a copy & paste error. See comment. Works for me.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (fabianmoronzirfas) [reply CODE_OF_CONDUCT.md]: Fixed.

**Comment by fabianmoronzirfas** [comment]
@ffd8 @trych are you okay with this? We need to move forward so we don't lose the OSS Team rights on netlify.

**Comment by fabianmoronzirfas** [comment]
To make this more prominent we could also create a repo for the CoC and just link there in each repo from a document like this one.

**Comment by trych** [comment]
> @ffd8 @trych are you okay with this?

Yes, sure.

**Comment by ffd8** [comment]
Sounds good, though I think it's fine to just have in the main basil.js repo (that's how p5.js is doing it, not for each project, web-editor or website)?

**Comment by fabianmoronzirfas** [comment]
That’s why I thought having a dedicated repo would be good

**Pull request merged**